### ISSUE TITLE
[gtk3] 3.0.2 does not start on Artful

### ISSUE THREAD

**Issue by egmontkob** [Guake/guake#1105]
I've happily used (tested) the Debian package 3.0.0.b2-1 on Artful.

With their new 3.0.2-1 package (some version requirements are slightly missed, I think the package is too strict on version numbers), as well as after manual installation (see #1104), I get pretty much the same errors (only the exact paths differ) when trying to start up Guake:

```
INFO:guake.guake_logging:Logging configuration complete
DEBUG:guake.guake_logging:Debug mode enabled
DEBUG:guake.main:DBus not running, starting it
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/dbus/bus.py", line 175, in activate_name_owner
    return self.get_name_owner(bus_name)
  File "/usr/lib/python3/dist-packages/dbus/bus.py", line 361, in get_name_owner
    's', (bus_name,), **keywords)
  File "/usr/lib/python3/dist-packages/dbus/connection.py", line 651, in call_blocking
    message, timeout)
dbus.exceptions.DBusException: org.freedesktop.DBus.Error.NameHasNoOwner: Could not get owner of name 'org.guake3.RemoteControl': no such name

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/lib/python3.6/dist-packages/guake/main.py", line 235, in main
    remote_object = bus.get_object(DBUS_NAME, DBUS_PATH)
  File "/usr/lib/python3/dist-packages/dbus/bus.py", line 241, in get_object
    follow_name_owner_changes=follow_name_owner_changes)
  File "/usr/lib/python3/dist-packages/dbus/proxies.py", line 248, in __init__
    self._named_service = conn.activate_name_owner(bus_name)
  File "/usr/lib/python3/dist-packages/dbus/bus.py", line 180, in activate_name_owner
    self.start_service_by_name(bus_name)
  File "/usr/lib/python3/dist-packages/dbus/bus.py", line 278, in start_service_by_name
    'su', (bus_name, flags)))
  File "/usr/lib/python3/dist-packages/dbus/connection.py", line 651, in call_blocking
    message, timeout)
dbus.exceptions.DBusException: org.freedesktop.DBus.Error.ServiceUnknown: The name org.guake3.RemoteControl was not provided by any .service files

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/bin/guake", line 10, in <module>
    sys.exit(exec_main())
  File "/usr/local/lib/python3.6/dist-packages/guake/main.py", line 344, in exec_main
    if not main():
  File "/usr/local/lib/python3.6/dist-packages/guake/main.py", line 239, in main
    instance = Guake()
  File "/usr/local/lib/python3.6/dist-packages/guake/guake_app.py", line 157, in __init__
    SCHEMA_DIR, Gio.SettingsSchemaSource.get_default(), False
GLib.Error: g-file-error-quark: Failed to open file “/usr/local/lib/python3.6/dist-packages/guake/data/gschemas.compiled”: open() failed: No such file or directory (4)
```

**Comment by Stibbons** [comment]
is it a packaging issue or a guake one?

**Comment by egmontkob** [comment]
Well, since the manual installation gives the same error as the package, I assume it's a guake one.

**Comment by Stibbons** [comment]
Should be fixed by #1106
